Makefile

```diff
@@ -13,7 +14,11 @@ bash:
 
 .PHONY: ci
 ci:
-	@$(BASH_TEST) "bin/rails db:setup db:migrate ci"
+	@$(BASH_TEST) "bin/rails db:setup db:migrate assets:precompile ci"
```

Comment:
Why is it necessary to call `assets:precompile` before the `ci` rake task?

Reply:
`assets:precompile` causes `webpacker:compile` to run which is needed to create `public/packs-test/manifest.json`

`public/packs-test` is ignored in the `.gitignore` which causes `public/packs-test/manifest.json` to not exist during test run which creates error below which can be seen in full at http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fgibct-data-service/detail/react/13/pipeline

```
  3) DashboardsController behaves like an authenticating controller when being visited and submitting credentials outputs an error message if not successful

     Failure/Error: <%= javascript_pack_tag 'application' %>

     ActionView::Template::Error:
       Webpacker can't find application in /src/gibct/public/packs-test/manifest.json. Possible causes:
       1. You want to set webpacker.yml value of compile to true for your environment
          unless you are using the `webpack -w` or the webpack-dev-server.
       2. webpack has not yet re-run to reflect updates.
       3. You have misconfigured Webpacker's config/webpacker.yml file.
       4. Your webpack configuration is not creating a manifest.
       Your manifest contains:
       {
       }
     .
     .
     .
     .
     .
     .
     # --- Caused by: ---
     # Webpacker::Manifest::MissingEntryError:
     #   Webpacker can't find application in /src/gibct/public/packs-test/manifest.json. Possible causes:
     #   1. You want to set webpacker.yml value of compile to true for your environment
     #      unless you are using the `webpack -w` or the webpack-dev-server.
     #   2. webpack has not yet re-run to reflect updates.
     #   3. You have misconfigured Webpacker's config/webpacker.yml file.
     #   4. Your webpack configuration is not creating a manifest.
     #   Your manifest contains:
     #   {
     #   }
     #   /usr/local/bundle/gems/webpacker-4.0.7/lib/webpacker/manifest.rb:79:in `handle_missing_entry'
```